plugins/scaffolder/src/components/fields/VirtualizedListbox.tsx

```diff
@@ -22,29 +22,40 @@ const renderRow = (props: ListChildComponentProps) => {
   return React.cloneElement(data[index], { style });
 };
 
+const OuterElementContext = React.createContext({});
+
+const OuterElementType = React.forwardRef<HTMLDivElement>((props, ref) => {
+  const outerProps = React.useContext(OuterElementContext);
+  return <div ref={ref} {...props} {...outerProps} />;
+});
+
 export const VirtualizedListbox = React.forwardRef<
   HTMLDivElement,
   { children?: React.ReactNode }
 >((props, ref) => {
-  const itemData = React.Children.toArray(props.children);
+  const { children, ...other } = props;
+  const itemData = React.Children.toArray(children);
   const itemCount = itemData.length;
 
   const itemSize = 36;
 
   const itemsToShow = Math.min(10, itemCount);
-  const height = Math.max(itemSize, itemsToShow * itemSize - 0.5 * itemSize);
```

Comment:
Think we need to bring this back right?

Reply:
As I said in the PR description : 

> I also updated the calculation of height of FixedSizeList because it was really strange with small lists (< 10 elements), now it just have a max size of 10 elements.

Before : 
![Capture d’écran 2024-08-12 à 09 15 03](https://github.com/user-attachments/assets/ca9ae4a0-9779-4d71-8c13-c7d27b3240a1)
![Capture d’écran 2024-08-12 à 09 15 12](https://github.com/user-attachments/assets/2ca2cf2d-5b67-439d-8510-47b46fb6d606)

The problem is mostly visible with a few elements

Now : 
![Capture d’écran 2024-08-12 à 09 26 20](https://github.com/user-attachments/assets/b77174e0-92da-401e-8d11-b0cdd1b516ad)
![Capture d’écran 2024-08-12 à 09 26 12](https://github.com/user-attachments/assets/a83897b6-e7ef-4cf6-8a55-a1fa01537189)

I can adjust it to keep the behavior when there is more than 10 elements but for me it seems a bit too much for "just" this :)  

Tell me what you think about it :)

Reply:
Apologies - yep OK - makes sense!